tests/performance_manager/test_performance_manager.py

```diff
@@ -511,7 +512,7 @@ def test_bad_empty_static_table() -> None:
     static_tables = get_table_objects()
     test_table = {"stop_times": static_tables["stop_times"]}
 
-    with pytest.raises(KeyError):
+    with pytest.raises(pyarrow.ArrowInvalid):
         load_parquet_files(test_table, "/tmp/FEED_INFO/timestamp=0000000000")
```

Comment:
New exception type since the column filtering is being down my pyarrow and not pandas.